Makefile

```diff
@@ -39,48 +39,61 @@ help:
 setup: .env .venv requirements.txt.log requirements-mysql.txt.log requirements-postgresql.txt.log  ## Sets up the project (installs dependencies etc.)
 
 
-requirements.txt.log: requirements.txt
```

Comment:
These 3 "requirements" targets were changed from installing via the `.txt` files, to install via the `.txt.freeze` files. This makes setting the project up consistent for all developers.